Tools/Create-Azure-Sentinel-Solution/examples/Watchlist_ExampleTemplate.json

```diff
@@ -0,0 +1,38 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "workspace": {
+            "defaultValue": "<Enter Log Analytics Workspace name>",
+            "type": "string",
+            "metadata": {
+                "description": "Workspace name for Log Analytics where Sentinel is setup"
+            }
+        },
+        "WatchlistId": {
+            "type": "string",
+            "minLength": 1,
+            "defaultValue": "[newGuid()]",
+            "metadata": {
+                "description": "Resource group to deploy solution resources"
+            }
+        }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.OperationalInsights/workspaces/providers/Watchlists",
+            "name": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',parameters('WatchlistId'))]",
+            "apiVersion": "2021-03-01-preview",
+            "properties": {
+                "description": "test description",
+                "displayName": "test watchlist display name",
+                "source": "CSV",
+                "provider": "Microsoft",
+                "numberOfLinesToSkip": 0,
+                "itemsSearchKey": "Username",
+                "rawContent": "Username,Description\ntestName,testDescription",
```

Comment:
Okay so we won't load a CSV file or anything, will expect them to put the encoded string of the CSV here

Reply:
Yes, in order to ensure the templates are usable as-is.